content/how-to/add-user-ad-saml/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Adding a BluBracket user with Azure AD SAML"
+description: "How to add a BluBracket user with Azure AD SAML"
+lead: "How to integrate with Azure AD SAML"
+date: 2022-01-12T02:48:57+00:00
+draft: false
+weight: 300
+toc: true
+resources:
+- src: azureactivedirectory.png
+- src: enterpriseapplications.png
+---
+
+This article covers the SAML 2.0 authentication requests and responses that BluBracket supports for Single Sign-On (SSO) using Azure Active Directory (Azure AD).
+
+BluBracket requires an email attribute in order to successful identify and authenticate users; BluBracket looks for this vaule from the SAML identity claim http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress.
```

Comment:
Could this be said more clearly if the document started with an overview of the logical relationship between Azure AD and BluBracket? Key points appear to be:

1. Users must be registered in both Azure AD and BluBracket in order to use BluBracket—removing a user from either will terminate their access to BluBracket
2. The email address in BluBracket must match the email address in Azure AD
3. Passwords and 2fa are managed in Azure ID